canonical/pgbouncer-operator
# Dpe 21 handle config change

## Issue
Charm config changes need to be applied to the pgbouncer application through pgbouncer.ini. This includes generating some variables for.

## Context 

## Release Notes
- Add default config to pgb library
- Add config-changed handler
- Added method to generate pgbouncer config values from charm config. 
- Documentation & QOL updates

## Testing
- Manual testing
  - Set config, check file is updated correctly
- Unit testing
- Integration testing
  - Set config, check file is updated correctly
